packages/dev/serializers/src/glTF/2.0/dataWriter.ts

```diff
@@ -1,11 +1,24 @@
+/* eslint-disable @typescript-eslint/naming-convention */
 /* eslint-disable babylonjs/available */
+import type { TypedArray } from "core/types";
 
 /** @internal */
 export class DataWriter {
     private _data: Uint8Array;
     private _dataView: DataView;
     private _byteOffset: number;
 
+    private _typedArrayToWriteMethod: Record<string, Function> = {
+        Int8Array: this.writeInt8.bind(this),
+        Uint8Array: this.writeUInt8.bind(this),
+        Uint8ClampedArray: this.writeUInt8.bind(this),
+        Int16Array: this.writeInt16.bind(this),
+        Uint16Array: this.writeUInt16.bind(this),
+        Int32Array: this.writeInt32.bind(this),
+        Uint32Array: this.writeUInt32.bind(this),
+        Float32Array: this.writeFloat32.bind(this),

```

Comment:
`bind` is a very slow function. It is slow to call and the resulting function returned is also slow. Use lambda syntax instead.